packages/amplify-category-api/amplify-plugin.json

```diff
@@ -12,10 +12,7 @@
     "rebuild",
     "remove",
     "update",
-    "help",
-    "import",
-    "generate-schema",
-    "update-secrets"
```

Comment:
Can we comment these lines with note so it is easy to enable when required?

Reply:
Can't do that here since it's a JSON.